stdlib/public/core/ArrayCast.swift

```diff
@@ -152,7 +156,7 @@ internal func _arrayConditionalBridgeElements<SourceElement, TargetElement>(
   let buf = _ContiguousArrayBuffer<TargetElement>(
     uninitializedCount: source.count, minimumCapacity: 0)
   
-  var p = buf.firstElementAddress
+  var p = buf.firstElementAddress!
```

Comment:
Right, I didn't want to use `subscriptBaseAddress` in ContiguousArray if the equivalent code in Slice couldn't.